<launch>

    <arg name="tsdf_file"/>
    <arg name="bbox_file"/>
    <arg name="dsg_file" default=""/>

    <node pkg="kimera_dsg_builder" type="room_visualizer_node" name="room_visualizer_node"
          args="-tsdf_file=$(arg tsdf_file) -bbox_file=$(arg bbox_file) -dsg_file=$(arg dsg_file)"
          output="screen">
    </node>

    <node pkg="rviz" type="rviz" name="rviz"
          args="-d $(find kimera_dsg_builder)/rviz/room_visualizer.rviz"/>

</launch>
